draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -146,12 +144,12 @@ Context exchange or pre-provisioning is out of scope of this document.
 
 {{Fig-archi}} represents the architecture for compression/decompression, it is
 based on {{RFC8376}} terminology. The Device is sending applications flows
-using IPv6 or IPv6/UDP protocols. These flows might be compressed by an Static
+using IPv6 or IPv6/UDP protocols. These flows might be compressed by a Static
 Context Header Compression Compressor/Decompressor (SCHC C/D) to reduce headers
 size and fragmented (SCHC F/R).  The resulting information is sent on a layer two
 (L2) frame to an LPWAN Radio Gateway (RGW) that forwards the frame to a Network
 Gateway (NGW). The NGW sends the data to a SCHC F/R for defragmentation, if
-required, then C/D for decompression which shares the same rules with the
+required, then to C/D for decompression. The C/D shares the same rules with the
```

Comment:
This is now conflicting with Dominique proposition, but I find yours more clear.